app/javascript/src/global/_common.scss

```diff
@@ -170,10 +170,14 @@ strong {
             @media only screen and (max-width: 767px) {
                 display: none !important;
             }
-        }   
+        }
     }
 }
 
 .ui.table.stackable thead {
     background: #212224;
-  }
\ No newline at end of file
+  }
+
+  .auction-domain-name {
```

Comment:
Could you check the indent here. Not sure it is needed.

Reply:
Checked & fixed